base/services/svchost/svchost.h

```diff
@@ -12,15 +12,12 @@
 #define WIN32_NO_STATUS
 #define WIN32_LEAN_AND_MEAN
 
+#include <dpfilter.h>
 #include <rpc.h>
+#include <svc.h>
 #include <ndk/rtlfuncs.h>
 #include <ndk/kdtypes.h>
```

Comment:
Just keep the original ordering:
```suggestion
#include <rpc.h>
#include <ndk/rtlfuncs.h>
#include <ndk/kdtypes.h>
#include <dpfilter.h>

#include <svc.h>
```

Reply:
Fixed, thanks.